.circleci/config.yml

```diff
@@ -218,6 +259,7 @@ workflows:
           filters:
             branches:
               only: master
+      - docs
```

Comment:
presumably this should be only on master?

Reply:
Updating to only on non-master, re: conversation above.

Reply:
Update: should run on all branches to ensure docs site is deployable.